kasa/aestransport.py

```diff
@@ -164,49 +169,31 @@ async def send_secure_passthrough(self, request: str):
         resp_dict = json_loads(response)
         return resp_dict
 
-    async def perform_login(self, login_request: Union[str, dict], *, login_v2: bool):
+    async def perform_login_for_version(self, *, login_v2: bool):
```

Comment:
```suggestion
    async def _perform_login_for_version(self, *, login_v2: bool):
```
So to my understanding this is about the `lv` in `mgt_encrypt_schm`? If yes, it's better to use an integer here in case new versions appear in the future.

This looks like an internal method too, so let's mark it as such.

Reply:
Done